web/src/click_handlers.js

```diff
@@ -636,6 +637,24 @@ export function initialize() {
         window.location.reload();
     });
 
+    // Scheduled Messages
+
+    $("body").on("click", ".scheduled-message-row .restore-overlay-message", (e) => {
+        let scheduled_msg_id = $(e.currentTarget)
+            .closest(".scheduled-message-row")
+            .attr("data-message-id");
+        scheduled_msg_id = Number.parseInt(scheduled_msg_id, 10);
+        scheduled_messages.edit_scheduled_message(scheduled_msg_id);
+    });
+
+    $("body").on("click", ".scheduled-message-row .delete-overlay-message", (e) => {
+        let scheduled_msg_id = $(e.currentTarget)
+            .closest(".scheduled-message-row")
+            .attr("data-message-id");
+        scheduled_msg_id = Number.parseInt(scheduled_msg_id, 10);
+        scheduled_messages.delete_scheduled_message(scheduled_msg_id);
+    });
```

Comment:
Generally we've been adding new click handlers in the module for the feature itself, I guess this would be `reminder.js`? This catch-all file is a thing we're trying to get rid of. 
Also this probably wants stopPropagation/preventDefault; most click handlers do.

Reply:
Okay, done.